Runner/Tests/TestConfiguration3.json

```diff
@@ -0,0 +1,22 @@
+{
+  "tests": [
```

Comment:
📓 Possible contextual rename required here as well.

Reply:
I talked with Ivan on this one, he agrees with 'tests'